runtime/battery-station/src/lib.rs

```diff
@@ -39,7 +39,7 @@ pub use crate::parachain_params::*;
 pub use crate::parameters::*;
 use alloc::vec;
 use frame_support::{
-    traits::{ConstU16, ConstU32, Contains, EnsureOneOf, EqualPrivilegeOnly, InstanceFilter},
+    traits::{ConstU16, ConstU32, Contains, EitherOfDiverse, EqualPrivilegeOnly, InstanceFilter},
     weights::{constants::RocksDbWeight, ConstantMultiplier, IdentityFee},
 };
```

Comment:
We should set `#![recursion_limit = "512"]` (`256 -> 512`) in all runtime crates as done by Substrate: https://github.com/paritytech/substrate/compare/polkadot-v0.9.23...polkadot-v0.9.26#diff-67684005af418e25ff88c2ae5b520f0c040371f1d817e03a3652e76b9485224aR22-R23